src/main/java/com/venafi/vcert/sdk/connectors/cloud/Cloud.java

```diff
@@ -3,84 +3,71 @@
-import com.venafi.vcert.sdk.connectors.cloud.domain.TagProjectZone;
-import com.venafi.vcert.sdk.connectors.cloud.domain.UserDetails;
-import com.venafi.vcert.sdk.utils.FeignUtils;
 
 public interface Cloud {
   @Headers("tppl-api-key: {apiKey}")
-  @RequestLine("GET /useraccounts")
+  @RequestLine("GET v1/useraccounts")
   UserDetails authorize(@Param("apiKey") String apiKey);
-
-  @Headers("tppl-api-key: {apiKey}")
-  @RequestLine("GET /projectzones/{zoneId}")
-  ProjectZone zoneById(@Param("zoneId") String zoneId, @Param("apiKey") String apiKey);
-
-  @Headers("tppl-api-key: {apiKey}")
-  @RequestLine("GET /projectzones/{zone}")
-  ProjectZone zones(@Param("zone") String zone, @Param("apiKey") String apiKey);
-
-  @Headers("tppl-api-key: {apiKey}")
-  @RequestLine("GET /devopsprojects?zoneDetails=true")
-  Projects projects(@Param("apiKey") String apiKey);
-
+  
   @Headers("tppl-api-key: {apiKey}")
-  @RequestLine("GET /devopsprojects/{projectId}?zoneDetails=true")
-  Project projectById(@Param("projectId") String projectId, @Param("apiKey") String apiKey);
-
-  @Headers("tppl-api-key: {apiKey}")
-  @RequestLine("GET /zones/tag/{tag}")
-  TagProjectZone zoneByTag(@Param("tag") String tag, @Param("apiKey") String apiKey);
-
+  @RequestLine("GET /outagedetection/v1/applications/{appName}/certificateissuingtemplates/{citAlias}")
+  CertificateIssuingTemplate certificateIssuingTemplateByAppNameAndCitAlias(
+	  @Param("appName") String appName,
+      @Param("citAlias") String citAlias,
+      @Param("apiKey") String apiKey);
+  
   @Headers("tppl-api-key: {apiKey}")
-  @RequestLine("GET /certificateissuingtemplates/{certificateIssuingTemplateId}")
-  CertificateIssuingTemplate certificateIssuingTemplateById(
-      @Param("certificateIssuingTemplateId") String certificateIssuingTemplateId,
+  @RequestLine("GET /outagedetection/v1/applications/name/{appName}")
+  Application applicationByName(
+	  @Param("appName") String appName,
       @Param("apiKey") String apiKey);
 
   @Headers({"tppl-api-key: {apiKey}", "Content-Type: application/json", "Accept: application/json"})
-  @RequestLine("POST /certificatesearch")
+  @RequestLine("POST /outagedetection/v1/certificatesearch")
   CertificateSearchResponse searchCertificates(@Param("apiKey") String apiKey,
       SearchRequest searchRequest);
 
   @Headers({"tppl-api-key: {apiKey}", "Content-Type: application/json"})
-  @RequestLine("POST /certificaterequests")
+  @RequestLine("POST  /outagedetection/v1/certificaterequests")
   CloudConnector.CertificateRequestsResponse certificateRequest(@Param("apiKey") String apiKey,
       CloudConnector.CertificateRequestsPayload csr);
 
   @Headers("tppl-api-key: {apiKey}")
-  @RequestLine("GET /certificaterequests/{id}")
+  @RequestLine("GET  /outagedetection/v1/certificaterequests/{id}")
   CertificateStatus certificateStatus(@Param("id") String id, @Param("apiKey") String apiKey);
 
   @Headers("tppl-api-key: {apiKey}")
```

Comment:
Are we missing the "Content-Type: text/plain" header here?

Reply:
Actually I forgot to remove that headers on:   Response certificateAsPem(@Param("id") String id, @Param("apiKey") String apiKey), now removed it on that so now we are consistent